.github/workflows/ci.yml

```diff
@@ -217,7 +217,15 @@ jobs:
           - mysql
           - trino
           - postgresql
-        python-version: ["3.8"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        exclude:
+          # TODO: would like to adopt `actionlint` pre-commit hook
+          # but false positive here and inability to do an inline ignore
+          # prevents this https://github.com/rhysd/actionlint/issues/237
+          - python-version: ${{ github.event_name != 'schedule' && '3.9' }}
+          - python-version: ${{ github.event_name != 'schedule' && '3.10' }}
+          - python-version: ${{ github.event_name != 'schedule' && '3.11' }}
```

Comment:
I tried multiple variations trying to get this to work in one line and none of them worked.
I've seen some other solutions on stack overflow but they seemed overly complicated and non-intuitive.